clang-tools-extra/clang-tidy/performance/MoveConstArgCheck.cpp

```diff
@@ -44,6 +44,12 @@ void MoveConstArgCheck::registerMatchers(MatchFinder *Finder) {
                unless(isInTemplateInstantiation()))
           .bind("call-move");
 
+  // Match ternary expressions where either branch contains std::move
+  auto TernaryWithMoveMatcher =
+      conditionalOperator(
+            hasDescendant(MoveCallMatcher)
+      ).bind("ternary-move");
```

Comment:
```suggestion
      );
```
bind isonly needed when you want to use it later.

Reply:
Amazing! Appreciate your feedback, all of your comments should be addressed by now.